opencti-platform/opencti-front/src/private/components/common/drawer/DrawerOpenCTI.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FunctionComponent } from 'react';
+import Drawer from '@mui/material/Drawer';
+import makeStyles from '@mui/styles/makeStyles';
```

Comment:
this is an excellent inititative!!!! 
makstyle seems to be [deprecated](https://mui.com/system/styles/basics/) (incompatible with React 18).
I don't fully understand the documentation and can't say whether it's very useful, but there seems to be a topic on the use of `makestyle` hook.

Reply:
Yep, deprecated, but we are not on React 18 yet and not prepared to make the jump. 
That being said, we could try to use the correct method from now on : import { styled } from "@mui/system";

Reply:
https://stackoverflow.com/questions/71948818/no-result-using-makestyles-material-ui-in-react-18